feconf.py

```diff
@@ -294,6 +294,10 @@
 
 # Default description for a newly-minted topic.
 DEFAULT_TOPIC_DESCRIPTION = ''
+# Default abbreviated name for a newly-minted topic.
+DEFAULT_ABBREVIATED_TOPIC_NAME = ''
+# Default thumbnail data url for a newly-minted topic.
+DEFAULT_THUMBNAIL_DATA_URL = ''
```

Comment:
This should be None, and it doesn't need to be defined as a constant here. None means "there's no URL yet"; a string means that it's a valid URL.

Reply:
Removed this.

Reply:
Done.